lib/std/fmt.zig

```diff
@@ -1713,3 +1726,19 @@ test "positional with specifier" {
 test "positional/alignment/width/precision" {
     try testFmt("10.0", "{0d: >3.1}", .{@as(f64, 9.999)});
 }
+
+test "vector" {
+    if (builtin.arch == .mipsel) return error.SkipZigTest;
```

Comment:
Please link to the github issue for this so we can find this code and enable the test when the issue is solved (by grepping for the issue number)